components/Login/LoginButton.module.css

```diff
@@ -0,0 +1,17 @@
+.login {
+  background-color: var(--spotify-color);
+
+  /* padding: 1em; */
+  border-radius: 50px;
+  border: none;
+  font-family: var(--primary-font);
+  font-size: 16pt;
```

Comment:
Use either px or pt - should be the same I think?

Reply:
Another option is to use relative sizes like `em` or `rem`.